mscore/inspector/inspector_tremolo.ui

```diff
@@ -150,7 +150,7 @@
         </item>
         <item>
          <property name="text">
-          <string>All lines attached to stem</string>
+          <string>All lines attached to stems</string>
```

Comment:
they are called beams elswhere

Reply:
If changing, I'd use "strokes" here and in other places of the same dialog

Reply:
Dorico calles them strokes, see https://steinberg.help/dorico_pro/v2/en/dorico/topics/notation_reference/notation_reference_tremolos_c.html

Reply:
Then let's call them strokes.